eslint-custom-rules/jest-mock-path-exists.js

```diff
@@ -0,0 +1,110 @@
+const fs = require('fs')
+const path = require('path')
+
+const pathExistsCache = new Map()
+
+function pathExists(filePath) {
+  if (pathExistsCache.has(filePath)) {
+    return pathExistsCache.get(filePath)
+  }
+
+  try {
+    fs.statSync(filePath)
+    pathExistsCache.set(filePath, true)
+    return true
+  } catch (error) {
+    pathExistsCache.set(filePath, false)
+    return false
+  }
+}
+
+function pathExistsWithExtensions(basePath, extensions) {
+  return extensions.some((ext) => pathExists(basePath + ext))
+}
+
+function validateMockPath(mockPath, context, extensions) {
+  if (extensions.some((ext) => mockPath.endsWith(ext))) {
```

Comment:
normalement, ça devrait fonctionner

c'est peut etre moins lisible, peut etre qu'il ne faut pas prendre cette suggestion

```suggestion
  if (extensions.some(mockPath.endsWith)) {
```